searx/engines/sogou.py

```diff
@@ -0,0 +1,71 @@
+# SPDX-License-Identifier: AGPL-3.0-or-later
+"""Sogou search engine for searxng"""
+
+from urllib.parse import urlencode
+from lxml import html
+
+# Metadata
+about = {
+    "website": "https://www.sogou.com/",
```

Comment:
```suggestion
    "website": "https://www.sogou.com/",
    "wikidata_id": "Q7554565"
```
ref https://www.wikidata.org/wiki/Q7554565